backend/pkg/accesslog/middleware_gin.go

```diff
@@ -34,15 +35,29 @@ type AccessLogger struct {
 	ClientIPHook func(r *http.Request) net.IP
 }
 
+func formatPathParams(c *gin.Context) string {
+	var b strings.Builder
+	l := len(c.Params) - 1
+	d := ","
+	for i, p := range c.Params {
+		if i == l {
+			d = ""
+		}
+		fmt.Fprintf(&b, "%s=%s%s", p.Key, p.Value, d)
```

Comment:
```suggestion
		fmt.Fprintf(&b, "%s=%s%s ", p.Key, p.Value, d)
```
? perhaps it would be nice? r you want it like that? take it or leave it.